### ISSUE TITLE
Use window.Vaadin.Flow instead of window.vaadin

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#3119]
Elements use `window.Vaadin.*` to publish classes and versions to JS, e.g. `window.Vaadin.ChartElement`. To be consistent across the platform, Flow should use `window.Vaadin.Flow` for its JS API instead of `window.vaadin`. 

This change would require at least some changes to TestBench `waitForVaadin` and possibly to some other integrations also.

**Comment by Legioth** [comment]
Note that Flow currently uses both `window.vaadin` and `window.flow`. Both should be folded into `window.Vaadin.Flow`.

**Comment by pleku** [comment]
Should be an easy fix with search and replacing (by hand). 🙈

**Comment by pleku** [comment]
I think we need a version of testbench that has this fixed before we can actually merge this change from Flow side, since TB is using automatically (should at least) `TestBench.waitForVaadin()`.

**Comment by SomeoneToIgnore** [comment]
TestBench pull request: https://github.com/vaadin/testbench/pull/971
Flow branch with changes: https://github.com/vaadin/flow/tree/kb/3119_window-vaadin-flow
All branch tests pass locally with TB version built locally.

After the TB PR is merged and new TB version is released, the Flow PR will be created.
